contracts/FT-voting-quadratic/RIFGovernorCountingQuadratic.sol

```diff
@@ -0,0 +1,179 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.9;
+
+import '@openzeppelin/contracts/governance/extensions/GovernorVotes.sol';
+
+import '../util/FixedPointMathLib.sol';
+
+/**
+ * @dev Extension of {Governor} for quadratic (sqrt), 3 options, vote counting.
+ * Iherit from GovernorVotes to have access to `token` in order to calculate quorum.
+ * `quorum` is defined here as prescibed in the Governor docs.
+ */
+abstract contract RIFGovernorCountingQuadratic is GovernorVotes {
```

Comment:
It's a pity that this could not be inherited and only override the target functions, but looks like the original from OZ was not designed for this use case.

Reply:
also, w.r.t. naming, this is not at all specific to the token being used as the governance FT, so we should not have `RIF` in the name. I think the previous name was fine.